lrmd/pacemaker_remote.in

```diff
@@ -48,9 +48,11 @@ failure()
 status()
 {
 	pid=$(pidof $1 2>/dev/null)
-	rtrn=$?
+	local rtrn=$?
 	if [ $rtrn -ne 0 ]; then
 		echo "$1 is stopped"
+		rm -rf "@localstatedir@/run/$prog.pid"
+		rtrn=3
```

Comment:
True, unfortunately I realized that it makes sense to go for pacemaker_remote as well when the commit for sbd in pacemaker.in was there already ... but of course I can take them apart again.